src/repartidor.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+type Repartidor struct {
+	id      int
+	horario Horario
+	pedidos Pedido
+}
+
+type Horario struct {
+	inicio string
+	fin    string
+}
+
+func NuevoRepartidor(id int, horario Horario) *Repartidor {
+	return &Repartidor{
+		id:      id,
+		horario: horario,
+	}
+}
+
+func (r *Repartidor) ID() int {
+	return r.id
+}
+
+func (r *Repartidor) Horario() Horario {
+	return r.horario
+}
+
+func (r *Repartidor) Pedidos() Pedido {
```

Comment:
Aquí, por tanto, también debería devolver el array:
```suggestion
func (r *Repartidor) Pedidos() []Pedido
```

Reply:
Aquí igual que la anterior, en el archivo repartidor.go nuevo está subsanado.